client/src/modules/Course/Components/ReviewModal.tsx

```diff
@@ -251,6 +232,9 @@ const ReviewModal = ({
               id="review-content"
               placeholder={placeholdertext}
             ></textarea>
+            <p className={styles.anonymouslabel}>
```

Comment:
<img width="590" alt="image" src="https://github.com/user-attachments/assets/920eb539-4fca-4d57-aad9-fcf08721a92b" />
Is it possible to make this text centered between the text box and the submit button?